presto-raptor/src/test/java/com/facebook/presto/raptor/storage/TestOrcFileRewriter.java

```diff
@@ -103,7 +111,7 @@ public void testRewrite()
         List<Type> columnTypes = ImmutableList.of(BIGINT, createVarcharType(20), arrayType, mapType, arrayOfArrayType, decimalType);
 
         File file = new File(temporary, randomUUID().toString());
-        try (FileWriter writer = new OrcRecordWriter(columnIds, columnTypes, file)) {
+        try (FileWriter writer = OrcTestingUtil.createFileWriter(columnIds, columnTypes, file, useOptimizedOrcWriter)) {
```

Comment:
Here we only test for `BIGINT`, `VARCHAR`, `ARRAY`, `MAP`, `ARRAY[ARRAY]` and `DECIMAL`. We should add more types to have full coverage on types that Raptor supports. At least we should add test cases for `TIME`, `TIMESTAMP`, `DATE`. For `TIMESTAMP WITH TIME ZONE`, since it's a bug, I suggest that we don't include it in any of the test...